scripts/emailings/reset-invalid-emails.php

```diff
@@ -67,37 +68,55 @@
 
 $db->begin();
 
+$myfile = fopen($id, "r") or die("Unable to open file\n");
+$groupofemails=array();
+for ($i=0; $i < MAXEMAILS; $i++)
+{
+	$tmp =fgets($myfile);
+	if ($tmp == null)
+	{
+		break;
+	}
+	$groupofemails[$i] = trim($tmp, "\n");
+}
```

Comment:
Here, you will process only MAXEMAILS entries.
Each loop process MAXEMAILS. So if there is 1000 lines in entry fails, we make 10 loops. So you must also introduce a more global loop that run until the end of file is not reached